components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/application/management/v1/AdvancedApplicationConfiguration.java

```diff
@@ -171,7 +172,24 @@ public void setEnableAuthorization(Boolean enableAuthorization) {
         this.enableAuthorization = enableAuthorization;
     }
 
+    /**
+     * Decides whether application is a shared application.
+     */
+    public AdvancedApplicationConfiguration sharedApp(Boolean sharedApp) {
```

Comment:
was this manually added? Ideally the swagger definition should be updated and the code should be regenerated

Reply:
Done.